db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_02_27_190524) do
+ActiveRecord::Schema.define(version: 2018_02_27_185947) do
```

Comment:
I'm curious if you know why this was needed? I'm a bit confused by it. I don't think it hurts anything but it also seems odd.

Reply:
Nope, I'm as confused as you are. Let's talk about it today.

Reply:
@JPrevost Am I clear to resolve this? My understanding is that we determined the timestamp is correct, despite its weirdness.

Reply:
Yeah I think we're good. I'll resolve the convo. Thanks for following up to confirm.